### ISSUE TITLE
Sending an attribute with a malformed syntax in `sort` at search time does not raise an invalid_sort error

### ISSUE THREAD

**Issue by gmourier** [meilisearch/MeiliSearch#1685]
With sortable-attributes

```json
["title", "rates"]
```

If I send `sort` in this way at search time (missing :asc/:desc)
```json
{
    "sort": ["title"]
}
```
or if I send `sort` in this way (malformed :asc/:desc)
```json
{
    "sort": ["title:deesc"]
}
```

The API response is 
```json
{
    "message": "invalid criterion title:deesc",
    "errorCode": "internal",
    "errorType": "internal_error",
    "errorLink": "https://docs.meilisearch.com/errors#internal"
}
```

As a user, I would like to receive an `invalid_sort` error indicating a syntax error in the `sort` parameter.

**Comment by irevoire** [comment]
One part of this issue has been fixed by https://github.com/meilisearch/milli/pull/342
But we still need to fix another part in Meilisearch.

https://github.com/meilisearch/MeiliSearch/blob/main/meilisearch-http/src/index/search.rs#L110-L114

**Comment by curquiza** [comment]
Closed by #1686
